autotest/gcore/vrt_read.py

```diff
@@ -1505,6 +1505,17 @@ def test_vrt_protocol():
     ds = gdal.Open("vrt://data/uint16_3band.vrt?exponent_2=2.2&scale_2=0,10,0,100")
     assert ds.GetRasterBand(2).Checksum() == 4455
 
+    ds = gdal.Open("vrt://data/float32.tif?outsize=10")
+    assert ds is None
```

Comment:
to avoid the GDAL error message to be displayed in verbose pytest runs:
```suggestion
    with gdaltest.error_handler():
        ds = gdal.Open("vrt://data/float32.tif?outsize=10")
        assert ds is None
```